### PR TITLE
refactor(app): update factory reset slideout for OT-3

### PR DESCRIPTION
**Pull request by brenthagen**
# Overview

localizes factory reset options and makes copy updates for ot-3. previously the reset option text labels were rendered directly from the api reset options response.

closes RCORE-229

# Changelog

 -  Updates factory reset slideout for OT-3

# Review requests

confirm correct copy for ot-2/ot-3

# Risk assessment

low

### CODE REVIEW

File: app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/FactoryResetSlideout.tsx [hunk 1]
```diff
@@ -68,8 +71,21 @@ export function FactoryResetSlideout({
...
 
-  const calibrationOptions =
+  const ot2CalibrationOptions =
     options != null ? options.filter(opt => opt.id.includes('Calibration')) : []
+  const ot3CalibrationOptions =
+    options != null
+      ? options.filter(
+          opt =>
+            opt.id === 'pipetteOffsetCalibrations' ||
+            opt.id === 'gripperCalibration'
```
Comment (koji) [inline app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/FactoryResetSlideout.tsx:81]: Seems that robot services team hasn't added a new option for gripper.
The option name for gripper calibration is fixed?

Comment (koji) [inline app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/FactoryResetSlideout.tsx:83]: One quick question
Seems the robot services team hasn't added a new option.
The option name for gripper calibration is fixed?

**Review by koji — APPROVED** [review]
Looks good to me!

File: app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/FactoryResetSlideout.tsx (hunk 1 above)
Comment (brenthagen) [reply app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/FactoryResetSlideout.tsx:83]: i'm making a guess here - i'll drop a TODO to update/confirm when the new option exists

**Pull request merged**